dpgen2/fp/vasp.py

```diff
@@ -37,7 +38,10 @@ def __init__(
         """
         self.kspacing = kspacing
         self.kgamma = kgamma
+        # TODO: not acctually optional
```

Comment:
I agree with you. Could you please remove the `Optional` of the `incar_template_name` and `potcar_names`? Thanks!